dataprep/tests/eda/test_report.py

```diff
@@ -1,71 +0,0 @@
-"""
```

Comment:
Why this file is deleted?

Reply:
Because the existing test scripts can cover the scenarios in `create_report`.

Reply:
okay

Reply:
I checked again the code coverage and it looks like these tests are necessary. The save and repr_html are not tested elsewhere.